src/Repository/SubmissionRepository.php

```diff
@@ -52,6 +54,31 @@ public function findFrontPageSubmissions(array $forums, string $sortBy, int $pag
     public function findForumSubmissions(Forum $forum, string $sortBy, int $page = 1) {
         $qb = $this->findSortedQb($sortBy)
             ->andWhere('s.forum = :forum')
+            ->andWhere('s.modThread = false')
+            ->setParameter('forum', $forum);
+
+        if ($sortBy === 'hot') {
+            PrependOrderBy::prepend($qb, 's.sticky', 'DESC');
+        }
```

Comment:
should this if-section be in separate PR/feature?

Reply:
This is just GIT being weird with diffs, this needs to be in here.

I duplicated this function but for the mod posts, and git got all weird